notNeededPackages.json

```diff
@@ -4182,10 +4182,6 @@
             "libraryName": "reapop",
             "asOfVersion": "3.0.0"
         },
-        "recharts": {
-            "libraryName": "recharts",
-            "asOfVersion": "2.0.0"
-        },
```

Comment:
so if the 2.0.0 types are still in the library, shouldn't we leave this?

Reply:
Hi! Thank you for your review!

I don't know how to do right. I tried to add only the v1 folder, but CI was failed because not found recharts/index.d.ts, so I added stub types for v2 and removed recharts from notNeededPackages.json.

Should I add recharts to notNeededPackages.json?